operating_unit/models/operating_unit.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import api, fields, models
+
+
+class OperatingUnit(models.Model):
+
+    _name = 'operating.unit'
+    _description = 'Operating Unit'
+
+    name = fields.Char('Name', size=256, required=True)
+    code = fields.Char('Code', size=32, required=True)
+    active = fields.Boolean('Active', default=True)
+    company_id = fields.Many2one(
+        'res.company', 'Company', required=True, default=lambda self:
+        self.env['res.company']._company_default_get('account.account'))
+    partner_id = fields.Many2one('res.partner', 'Partner', required=True)
+
```

Comment:
does partner_id is the manager of the OU? In that case changing the label to OU manager or just manager could be more meaningful...